src/accordionpane/tests/unit/AccordionPane.ts

```diff
@@ -0,0 +1,95 @@
+import { v, w } from '@dojo/widget-core/d';
+import * as assert from 'intern/chai!assert';
+import * as registerSuite from 'intern!object';
+import * as sinon from 'sinon';
+import harness, { Harness } from '@dojo/test-extras/harness';
+
+import AccordionPane, { AccordionPaneProperties } from '../../AccordionPane';
+import TitlePane from '../../../titlepane/TitlePane';
+
+import * as css from '../../styles/accordionPane.m.css';
+
+let pane: Harness<AccordionPaneProperties, typeof AccordionPane>;
+registerSuite({
+	name: 'AccordionPane',
+
+	beforeEach() {
+		pane = harness(AccordionPane);
+	},
+
+	afterEach() {
+		pane.destroy();
+	},
+
+	'default rendering'() {
+		pane.expectRender(v('div', {
+			classes: pane.classes(css.root)
+		}, []));
+	},
+
+	'default rendering with children'() {
+		pane.setProperties({ openKeys: [] });
+
+		pane.setChildren([
+			w(TitlePane, { title: 'foo', key: 'foo', onRequestOpen: () => {} }),
+			undefined,
+			w(TitlePane, { title: 'bar', key: 'bar', onRequestClose: () => {} }),
+			w(TitlePane, { title: 'baz', key: 'baz' })
+		]);
+
+		pane.expectRender(v('div', {
+			classes: pane.classes(css.root)
+		}, [
+			w(TitlePane, {
+				key: 'foo',
+				onRequestClose: pane.listener,
+				onRequestOpen: pane.listener,
+				open: false,
+				theme: undefined,
+				title: 'foo'
+			}),
+			w(TitlePane, {
+				key: 'bar',
+				onRequestClose: pane.listener,
+				onRequestOpen: pane.listener,
+				open: false,
+				theme: undefined,
+				title: 'bar'
+			}),
+			w(TitlePane, {
+				key: 'baz',
+				onRequestClose: pane.listener,
+				onRequestOpen: pane.listener,
+				open: false,
+				theme: undefined,
+				title: 'baz'
+			})
+		]));
+	},
+
+	'onRequestOpen should be called'() {
```

Comment:
Should there be tests to show that children created with/without `onRequest*` have it called when present, and that the accordion pane has `onRequest*` called in either case?